packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -106,7 +106,7 @@
       "editor/context": [
         {
           "command": "sfdx.force.lightning.lwc.preview",
-          "when": "sfdx:project_opened && resource =~ /.*/lwc/[^/]+(/[^/]+\\.(html|css|js|xml))?$/"
+          "when": "sfdx:project_opened && resource =~ /.*/lwc/[^/]+(/[^/]+\\.(html|css|js))?$/"
```

Comment:
this will have to be fixed on the lsp utility side then we can add support back for right clicking on meta xml files